CHANGES.md

```diff
@@ -1,13 +1,15 @@
 ## Changelog
 
 ### 1.70
+  * Bugfix: #157 Preserve NaN and don't blindly stringify columns that don't only contain strings
```

Comment:
There's a lot of "don't"s in here making it a bit difficult to read.

Maybe this instead?
```
* Bugfix: #157 Preserve NaNs in columns containing strings
```

Reply:
yeah, language is not great, but this fix is more generic than NaNs, will change it to smth like:

Bugfix: #157 Assure that serialized dataframes remain value-equivalent (e.g. preserve NaNs in mixed string columns)

Reply:
don